### PR TITLE
unibilium 1.2.0

### PR DESCRIPTION
**Pull request by dholm**

### CODE REVIEW

File: Library/Formula/unibilium.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+class Unibilium < Formula
+  desc "Unibilium is a very basic terminfo library."
```
Comment (bfontaine) [inline Library/Formula/unibilium.rb]: You can remove the name from the `desc`: `Very basic terminfo library`.

**Comment by dholm** [comment]
@bfontaine Fixed!

File: Library/Formula/unibilium.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+class Unibilium < Formula
+  desc "A very basic terminfo library."
```
Comment (DomT4) [inline Library/Formula/unibilium.rb]: `* Please remove the indefinite article "A" from the beginning of the description`

**Comment by xu-cheng** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

**Pull request closed without merging**